spec/integration_spec.rb

```diff
@@ -116,13 +120,14 @@
                             '--root-url https://realm.io/docs/swift/' \
                             "#{realm_version}/api/ " \
                             '--xcodebuild-arguments ' \
-                            '-project,RealmSwift.xcodeproj,-dry-run'
+                            '-project,RealmSwift.xcodeproj,-dry-run ' \
+                            '--template-directory "docs/templates/swift" '
     end
 
-    describe 'Creates docs for a podspec with dependencies and subspecs' do
-      behaves_like cli_spec 'document_moya_podspec',
-                            '--podspec=Moya.podspec'
-    end
+    # describe 'Creates docs for a podspec with dependencies and subspecs' do
```

Comment:
Looks like (for Moya, anyway) the old Swift 1.2 integration spec works without any modifications other than specifying `--swift-version=1.2`. :tada: